server/app/models/user.py

```diff
@@ -42,6 +56,9 @@ class User(Document):
     last_name = StringField(max_length=200, required=True)
     email = EmailField(unique=True, required=True)
     hashed_password = StringField(required=True)
+    tags = ListField(ReferenceField("Tag"))
```

Comment:
Now I know how [it](https://github.com/softwareVirus/micro-blog/pull/24#discussion_r1538457237) works :D 
But I'm not sure if this is the best way of delivering these related changes